### DISCUSSION TITLE
What is the current status of star imports?

### DISCUSSION THREAD

**Discussion by Tal500** [astral-sh/ruff#8569] [General]
I look up for a Pylance equivalent for correctly scanning definitions of star import.
I saw some issue in this repo about it, but I don't understand clearly ruff view on start import: do you scan star import definitions? Is it planned to be?

**Comment by zanieb** [comment]
We don't support analysis across multiple files yet, so if you use `from foo import *` we cannot know what names you've imported from `foo`. We usually include a message indicating this. We are interested in supporting this eventually.

**Reply by Tal500** [reply]
Thanks, this feature eventually will be very useful, and will put this linter above all the other ones available (not only in performance)
